pyanaconda/ui/gui/spokes/network.py

```diff
@@ -1019,8 +1019,10 @@ def _refresh_header_ui(self, dev_cfg, state=None):
 
         switch = self.builder.get_object("device_%s_off_switch" % dev_type_str)
         if dev_type_str == "wired":
-            switch.set_visible(state not in (NM.DeviceState.UNAVAILABLE,
-                                             NM.DeviceState.UNMANAGED))
+            visible = state not in (NM.DeviceState.UNAVAILABLE, NM.DeviceState.UNMANAGED)
+            switch.set_visible(visible)
+            if not visible:
+                switch.set_no_show_all(True)
```

Comment:
Shouldn't it be `switch.set_no_show_all(not visible)`? Is it a one-time thing?

Reply:
I guess the call with "reversed" argument is to be avoided?

Reply:
Modified according to @poncovka's suggestion. Also the rhel-8 branch PR.